services/web/server/src/simcore_service_webserver/catalog/_api_units.py

```diff
@@ -11,11 +11,20 @@
 
 
 def _get_type_name(port: BaseServiceIOModel) -> str:
-    _type: str = port.property_type
+    _type_name: str = port.property_type
```

Comment:
since you renamed the variable, I guess it does not need to be private anymore right?